src/xtopcom/xkad/src/elect_routing_table.cc

```diff
@@ -177,6 +177,16 @@ void ElectRoutingTable::GetRandomNodes(std::vector<NodeInfoPtr> & vec, size_t si
         }
     }
 }
+NodeInfoPtr ElectRoutingTable::GetOneRandomNode() {
```

Comment:
can be a const API?

Reply:
no, m_nodes_for_shuffle will realigned each time called